hooks/persistence-defectdojo/hook/src/test/java/io/securecodebox/persistence/service/ScanServiceTest.java

```diff
@@ -28,9 +30,10 @@ public class ScanServiceTest {
   @Mock
   Scan scan;
 
-  public static String readResourceAsString(String resourceName) throws IOException {
+  public static String readResourceAsString(String resourceName) throws IOException, URISyntaxException {
     ClassLoader cl = ScanServiceTest.class.getClassLoader();
-    File resourceFile = new File(cl.getResource(resourceName).getFile());
+    URI resourceURI = cl.getResource(resourceName).toURI();
+    File resourceFile = new File(resourceURI);
```

Comment:
It is not necessary to use file here. You can directly convert URI to Path: `Path.get(uri)`.